Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -2234,7 +2234,7 @@ WebsiteDataStoreParameters WebsiteDataStore::parameters()
         createHandleFromResolvedPathIfPossible(parentBundleDirectory(), parentBundleDirectoryExtensionHandle, SandboxExtension::Type::ReadOnly);
         parameters.parentBundleDirectoryExtensionHandle = WTFMove(parentBundleDirectoryExtensionHandle);
 
-        if (auto handleAndFilePath = SandboxExtension::createHandleForTemporaryFile(emptyString(), SandboxExtension::Type::ReadWrite))
+        if (auto handleAndFilePath = SandboxExtension::createHandleForTemporaryFile("com.apple.WebKit.Networking"_s, SandboxExtension::Type::ReadWrite))
```

Comment:
Is there a way to get the string from service-name instead of hard coding it? I think TestRunner might use "com.apple.WebKit.Networking.Development" on iOS simulator.
At least we probably want some test or assert to ensure UI process grants extension to a path that child process expects.